Versions.props

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
What is this file for? If this is meant to be the `corefx` version, this isn't correct, right?

Reply:
mistake I removed them https://github.com/dotnet/corefx/pull/32402.

Reply:
Sorry the VersionPrefix doesn't imply and will be cleaned up. The Version file being in the root was a mistake I fixed.